src/tests/FunctionalTests/Android/Device_Emulator/gRPC/grpc-dotnet/testassets/Certs/InteropTests/README.md

```diff
@@ -0,0 +1,8 @@
+Keys taken from https://github.com/grpc/grpc/tree/master/src/core/tsi/test_creds
+so that interop server in this project is compatible with interop clients
+implemented in other gRPC languages.
+
+The server1.pem and server1.key were combined into server1.pfx. The password is 1111. These certs are not secure, do not use in production.
```

Comment:
`The password is 1111` will most likely be flagged by credscan as well, you can use the magic string `PLACEHOLDER` as the password to avoid it: https://strikecommunity.azurewebsites.net/articles/8148/credscan-false-positives-and-suppressions.html#:~:text=PLACEHOLDERS